docs/_layouts/tutorial_frame.html

```diff
@@ -12,18 +12,19 @@
     <link rel="stylesheet" href="https://unpkg.com/leaflet@{{ site.latest_leaflet_version}}/dist/leaflet.css" integrity="{{site.integrity_hash_css}}" crossorigin=""/>
     <script src="https://unpkg.com/leaflet@{{ site.latest_leaflet_version}}/dist/leaflet.js" integrity="{{site.integrity_hash_uglified}}" crossorigin=""></script>
 
-{% unless page.customMapContainer == "true" %}
 	<style>
 		html, body {
 			height: 100%;
 			margin: 0;
 		}
-		#map {
-			width: 600px;
+		.leaflet-container {
 			height: 400px;
+			width: 600px;
+			max-width: 100%;
+			max-height: 100%;
 		}
 	</style>
-{% endunless %}
```

Comment:
Removing the condition, because pages with custom map containers can easily override this (and some do).